libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -1724,6 +1732,36 @@ void AC_AutoTune_Heli::updating_max_gains(float *freq, float *gain, float *phase
 
 }
 
+float AC_AutoTune_Heli::target_angle_max_rp_cd() const
```

Comment:
we could copy the comments from the .h to here in the .cpp.  It's not a blocker of course.